sys/sys/sysctl.h

```diff
@@ -1060,13 +1060,21 @@ TAILQ_HEAD(sysctl_ctx_list, sysctl_ctx_entry);
 #define	KERN_PROC_NFDS		43	/* number of open file descriptors */
 #define	KERN_PROC_SIGFASTBLK	44	/* address of fastsigblk magic word */
 #define	KERN_PROC_VM_LAYOUT	45	/* virtual address space layout info */
-#define	KERN_PROC_RLIMIT_USAGE	46	/* array of rlim_t */
+#define	KERN_PROC_RLIMIT_USAGE	51	/* array of rlim_t */
```

Comment:
I think I would rather renumber the CheriBSD ones now and just deal with the ABI fallout so that we minimize future diffs.  It's also not clear to me why we (including upstream) are adding any new sysctls with hardcoded values vs using sysctlnametomib() to look up the name.

Reply:
I think breaking CheriBSD's ABI is fine in this case.

I guess we should be converting code to use sysctlnametomib() to fill out the mib arrays instead of using the hard coded values.

Reply:
Does that work for MIBs that embed parameters like the target process PID in the OID?

Reply:
Yes, you can do `sysctlnametomib("kern.proc.foo")` to look up the first 3 members of the `mib[]` and then add the pid in the 4th member and use that to call sysctl().  There is in fact an example of this exact case for `kern.proc.pid` in the sysctl(3) manpage.

Reply:
Oh right, I forgot that the parameter comes at the end. I assumed it was something like <kern>.<proc>.<pid>.<foo>.